src/views/choose-general-partner-usual-residential-address.njk

```diff
@@ -1,7 +1,7 @@
 {% extends "layout.njk" %}
 
 {% set pageTitle = i18n.address.chooseAddress.generalPartnerUsualResidentialAddress.title %}
-{% set cache = props.data.cache['general_partner_usual_residential_address'] %}
+{% set cache = props.data.cache['usual_residential_address'] %}
```

Comment:
Assume that something was broken due to this incorrect reference?

Reply:
yes, it wouldn't display the cached data on the page due to the cache key changing to usual_residential_address